bonus_file_with_defs.rb

```diff
@@ -56,18 +59,27 @@ def main_menu(input)
     #####
   elsif input == "4"
     own_wordlist
-    begin
-      raise "Don't go to the next time in this case!"
-    rescue StandardError => input4
-      return 0
-    end
+    # begin
+    #   raise "Don't go to the next time in this case!"
+    # rescue StandardError => input4
+    #   return 0
+    # end
     #####
```

Comment:
Maybe it's not the best idea to depart code in such way. Use methods, modules and Classes.